### PR TITLE
style(front): add a welcome banner in the signup form

### PR DESCRIPTION
**Pull request by GresilleSiffle**
**related to** #1739 

---

This PR adds a welcome banner in the sign up form to make it more attractive.

In addition, a `helperText` has been added to the email field, to inform the users that an activation email will be sent to finalize the sign up process. For now there is no indication about the trusted / untrusted domains. I'm still trying to find the correct wording.

Finally, a security net has been added in the `SignupSuccess` component, to invite the users to check the spelling of their email address, and to create a new account in case of need (see the preview below).

#### Desktop

capture

#### Mobile

capture

#### Success

capture

### CODE REVIEW

**Comment by GresilleSiffle** [comment]
**to-do**
- [ ] update the e2e tests

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -889,10 +889,14 @@
...
-    "welcome": "Welcome!",
-    "successMessage": "A verification link has been sent to <1>{{email}}</1> .",
+    "oneLastStep": "One last step",
+    "successMessage": "A verification link has been sent to <1>{{email}}</1>",
+    "ifYourEmailIsIncorrect": "If your email address is incorrect, simply <2>create a new account</2>.",
+    "welcomeOnTournesol": "Welcome on Tournesol",
+    "weVeBeenWaitingForYou": "We've been waiting for you 🖖",
     "title": "Account creation",
     "accountCreationFailed": "Account creation failed.",
+    "anActivationEmailWillBeSent": "An activation email will be sent.",
```
Comment (amatissart) [inline frontend/public/locales/en/translation.json:899]:
```suggestion
    "anActivationEmailWillBeSent": "An activation email will be sent to confirm the address, before you can start contributing.",
```

File: frontend/public/locales/en/translation.json [hunk 2]
```diff
@@ -889,10 +889,14 @@
   "noVideoCorrespondsToSearchCriterias": "No video corresponds to your search criteria.",
   "profile": "Profile",
   "signup": {
-    "welcome": "Welcome!",
-    "successMessage": "A verification link has been sent to <1>{{email}}</1> .",
+    "oneLastStep": "One last step",
+    "successMessage": "A verification link has been sent to <1>{{email}}</1>",
+    "ifYourEmailIsIncorrect": "If your email address is incorrect, simply <2>create a new account</2>.",
+    "welcomeOnTournesol": "Welcome on Tournesol",
```
Comment (amatissart) [inline frontend/public/locales/en/translation.json:895]:
```suggestion
    "welcomeOnTournesol": "Welcome to Tournesol",
```

**Review by amatissart — COMMENTED** [review]
LGTM

A few suggestions, to translate in french if accepted.

**Review by amatissart — APPROVED** [review]

**Pull request merged**